akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/aa/AABlogExampleSpec.scala

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.akka.persistence.typed.aa
+
+import java.util.UUID
+
+import akka.Done
+import akka.actor.testkit.typed.scaladsl.{ LogCapturing, ScalaTestWithActorTestKit }
+import akka.actor.typed.ActorRef
+import akka.actor.typed.scaladsl.{ ActorContext, Behaviors }
+import akka.persistence.query.journal.leveldb.scaladsl.LeveldbReadJournal
+import akka.persistence.typed.scaladsl._
+import akka.serialization.jackson.CborSerializable
+import com.typesafe.config.ConfigFactory
+import org.scalatest.concurrent.{ Eventually, ScalaFutures }
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.time.{ Millis, Span }
+import org.scalatest.wordspec.AnyWordSpecLike
+
+object AABlogExampleSpec {
+  val config =
+    ConfigFactory.parseString(s"""
+                                            
+       akka.actor.allow-java-serialization = true 
+       // FIXME serializers for replicated event or akka persistence support for metadata: https://github.com/akka/akka/issues/29260
+       akka.actor.provider = cluster
+       akka.loglevel = debug
+       akka.persistence {
+        journal {
+        plugin = "akka.persistence.journal.leveldb"
```

Comment:
start using the new inmem journal instead?
https://doc.akka.io/docs/akka/current/typed/persistence-testing.html#persistence-testkit

Reply:
We can add support for the read side while doing this as it would be much better to use that

Reply:
I'll do that as a separate PR